src/Data/IdGenerator/Masa.Contrib.Data.IdGenerator.Snowflake.Distributed.Redis/Internal/SnowflakeExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Data.IdGenerator.Snowflake.Distributed.Redis.Internal;
+
+internal static class SnowflakeExtensions
+{
+    public static long GetTimestamp(this DateTimeOffset dateTimeOffset, uint timeType)
+    {
+        if (timeType == 1)
```

Comment:
What's the mean of the timeType?

Reply:
The timestamp in Snowflake ID is milliseconds by default. By changing the timestamp type to seconds, the number of bits occupied by the timestamp can be reduced. For some scenarios with low concurrency, the second-level timestamp can be used, and the performance will be higher.

Reply:
I mean you can use enum.

Reply:
ok  processed